BrickController2/BrickController2/PlatformServices/GameController/GameControllerEventArgs.cs

```diff
@@ -6,15 +6,19 @@ namespace BrickController2.PlatformServices.GameController
 {
     public class GameControllerEventArgs : EventArgs
     {
-        public GameControllerEventArgs(GameControllerEventType eventType, string eventCode, float value)
+        public readonly string ControllerDeviceId;
+
+        public GameControllerEventArgs(string controllerDeviceId, GameControllerEventType eventType, string eventCode, float value)
```

Comment:
Nit pick: I guess there should be constructor without id as well so as e.g. IOS should keep the gamepad code untouched.

Reply:
I think IOS also could have this feature (if someone just implements the distinction of the input events) - so there will be no need for a constuctor without id.